velox/functions/prestosql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -475,32 +475,34 @@ TEST_F(DateTimeFunctionsTest, yearDate) {
 }
 
 TEST_F(DateTimeFunctionsTest, yearTimestampWithTimezone) {
```

Comment:
Why we need this change? Thanks!

Reply:
We don't need this change. Some unrelated changes slips through. I'll delete them